lighthouse-core/gather/driver.js

```diff
@@ -400,7 +400,20 @@ class Driver {
    */
   async getAppManifest() {
     this.setNextProtocolTimeout(3000);
-    const response = await this.sendCommand('Page.getAppManifest');
+    let response;
+    try {
+      response = await this.sendCommand('Page.getAppManifest');
+    } catch (err) {
+      if (err.code === 'PROTOCOL_TIMEOUT') {
+        // LR will timeout fetching the app manifest in some cases, move on without one.
+        // https://github.com/GoogleChrome/lighthouse/issues/7147#issuecomment-461210921
+        log.error('Driver', 'Failed fetching...', err);
```

Comment:
ha, sorry, I meant ... for the rest of the message you had before, I just didn't want to retype it :)